.semaphore/semaphore.yml

```diff
@@ -73,6 +73,18 @@ blocks:
             - checkout
             - cache restore node-$(checksum yarn.lock)
             - yarn eslint workspaces/*/cypress
+  - name: Typescript Code Style Check
+    dependencies:
+      - Eslint
+    task:
+      jobs:
+        - name: typecodestype
```

Comment:
A small typo here maybe? I think you meant sth like "typecodestyle"? I would maybe just put "types" there

Reply:
fixed